src/bin/zmmailboxdctl

```diff
@@ -59,11 +59,31 @@ mk_download_dir() {
   fi
 }
 
+#
+# License check
+#
+check_license_status() {
+  local command="zmjava com.zimbra.cs.license.LicenseCLI -c"
```

Comment:
Please check with Rama he had introduced `forceQuit` options for the zmlicense.

Reply:
@chitranshu198 @RakeshAMore I think LicenseCLI  not available for FOSS edition. What will happen if mailbox is running with expired license?